Common/src/main/java/me/egg82/tfaplus/services/InternalAPI.java

```diff
@@ -161,24 +151,32 @@ public String registerTOTP(UUID uuid, long codeLength, JedisPool redisPool, Conf
         }
 
         // Redis
-        Redis.update(result, redisPool, redisConfigNode);
+        Redis.update(result, ConfigUtil.getRedisPool(), ConfigUtil.getRedisConfigNode());
 
         // RabbitMQ
-        RabbitMQ.broadcast(result, rabbitConnection);
+        try {
+            RabbitMQ.broadcast(result, getRabbitConnection());
```

Comment:
Not using the broadcast method, here?
Redis & Rabbit broadcast/update should probably be in one method, if we're going to make it a method anyway.